ui/src/router/components/configuration/components/section.scss

```diff
@@ -1,5 +1,13 @@
 @import "~@elastic/eui/src/global_styling/mixins/typography";
 
+// When text is truncated in a child flex element, the auto-calculated min width is wider than
+// its parent container which can break the layout. To prevent this, the min-width attribute
+// is expected to be set to a fixed value.
+// Ref: https://css-tricks.com/flexbox-truncated-text/
+.euiFlexItem {
+  min-width: 0;
+}
```

Comment:
Despite adding this to `ui/src/router/components/configuration/components/section.scss`, this style would be applied to all of the `.euiFlexItem` everywhere in the app, which can potentially lead to unexpected side effects. 

Would be better to at least nest it under `.euiPanel--configSection` or `euiDescriptionList`

Reply:
Thanks for pointing it out. I don't think this can be nested under `.euiPanel--configSection` or `euiDescriptionList` because the style must be applied to the FlexItem that encompasses the ConfigSection. There are many `EuiFlexItem`s in the DOM tree and from playing around, it seems like this property must be set on the second and third level flex items. I added a separate class to these components now.

<img width="650" alt="Screenshot 2020-11-12 at 6 16 05 PM" src="https://user-images.githubusercontent.com/23465343/98927378-6c820a00-2513-11eb-8d90-b197b972f148.png">